selfdrive/test/test_car_models.py

```diff
@@ -482,6 +487,7 @@ def get_route_log(route_name):
   "c9d60e5e02c04c5c|2018-01-08--16-01-49",  # HONDA CR-V 2016 TOURING
   "1632088eda5e6c4d|2018-06-07--08-03-18",  # HONDA CIVIC HATCHBACK 2017 SEDAN/COUPE 2019
   "fbd011384db5e669|2018-07-26--20-51-48",  # TOYOTA CAMRY HYBRID 2018
+  "a859a044a447c2b0|2020-02-21--17-51-36",  # HONDA CR-V 2016 EXECUTIVE
```

Comment:
Can you remove it from this list, otherwise the test doesn't run

Reply:
I have done this and a rebase. However, it looks like the test failed again. Maybe the route didn't make it to the bucket before it expired? I've changed it out for another.

Reply:
I made the drive as of commit 4ebbd03 public in useradmin. It was created on the 3rd.

Reply:
@pd0wm you see anything obvious why it isn't able to download the route for CRV_EU?